lib/components/viewers/trip-viewer.js

```diff
@@ -1,29 +1,30 @@
 // FIXME: Remove this eslint rule exception.
 /* eslint-disable jsx-a11y/label-has-for */
+import { Button, Label } from 'react-bootstrap'
+import { connect } from 'react-redux'
+import { FormattedMessage, FormattedTime } from 'react-intl'
 import coreUtils from '@opentripplanner/core-utils'
 import moment from 'moment'
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
-import { Button, Label } from 'react-bootstrap'
-import { FormattedMessage, FormattedTime } from 'react-intl'
-import { connect } from 'react-redux'
 
+import { findGraphQLTrip, findTrip } from '../../actions/api'
+import { setLocation } from '../../actions/map'
+import { setViewedTrip } from '../../actions/ui'
```

Comment:
Use the following syntax for importing actions that was adopted some time in 2020.
It prevents from accidentally calling an action without mapping it to dispatch:
```js
import * as apiActions from '../../actions/api'
...
const mapDispatchToProps = {
  findTrip: apiActions.findTrip
  ...
}

```

Reply:
That's why we do that! Genius. Refactored in ca08debd338383fb327a3543108c92b3e269a740